[2016-05-02 23:57:09,078] {models.py:126} INFO - Filling up the DagBag from /Users/bruno/Development/Intelivix/etl-example/dags/extract_judges.py
[2016-05-02 23:57:09,079] {models.py:204} INFO - Importing /Users/bruno/Development/Intelivix/etl-example/dags/extract_judges.py
[2016-05-02 23:57:09,294] {models.py:296} INFO - Loaded DAG <DAG: extract_judges>
[2016-05-02 23:57:14,359] {models.py:126} INFO - Filling up the DagBag from /Users/bruno/Development/Intelivix/etl-example/dags/extract_judges.py
[2016-05-02 23:57:14,360] {models.py:204} INFO - Importing /Users/bruno/Development/Intelivix/etl-example/dags/extract_judges.py
[2016-05-02 23:57:14,684] {models.py:296} INFO - Loaded DAG <DAG: extract_judges>
[2016-05-02 23:57:14,757] {models.py:1001} INFO - 
--------------------------------------------------------------------------------
Attempt 1 out of 2
--------------------------------------------------------------------------------

[2016-05-02 23:57:14,775] {models.py:1041} INFO - Executing <Task(PythonOperator): extract_name_task> on 2016-04-19 00:00:00
[2016-05-02 23:57:14,805] {python_operator.py:66} INFO - Done. Returned value was: None
